x-pack/plugins/fleet/public/applications/fleet/sections/agents/components/agent_health.tsx

```diff
@@ -27,7 +27,7 @@ const Status = {
     </EuiBadge>
   ),
   Inactive: (
-    <EuiBadge color="default">
+    <EuiBadge color="#98A2B3">
```

Comment:
using the same color as on status bar, trying to find out if there is an eui variable to reference instead

https://github.com/elastic/kibana/blob/master/x-pack/plugins/fleet/public/applications/fleet/sections/agents/services/agent_status.tsx#L22

Reply:
I think this color is the `euiColorDarkShade` in our theme

Reply:
@nchaulet thanks, though it is not the same color, should I change the status bar too to use dark shade instead?
![image](https://user-images.githubusercontent.com/90178898/135977179-77d5ba4e-a3bf-407e-a10e-1e7bc58f8bb3.png)

Reply:
I think we should use the EUI theme color here.

Reply:
useTheme was giving me an error from react hooks, I managed to import another way, please check if it's good
updated the theme variable to dark shade, it looks quite similar to the custom color before
![image](https://user-images.githubusercontent.com/90178898/136036321-59615fe8-d252-4acf-8d8d-f3a11cdba672.png)